bundles/org.openhab.binding.icalendar/src/main/resources/OH-INF/i18n/icalendar_de.properties

```diff
@@ -28,6 +28,11 @@ thing-type.config.icalendar.calendar.username.label = Benutzername
 thing-type.config.icalendar.calendar.username.description = Benutzername zum Abruf der Kalenderdaten (für Basic Authentication, nur sinnvoll mit gesetztem Passwort).
 thing-type.config.icalendar.eventfilter.datetimeEnd.label = Ende
 thing-type.config.icalendar.eventfilter.datetimeEnd.description = Endzeitpunkt relativ zu "jetzt" (exklusiv).
+thing-type.config.icalendar.eventfilter.datetimeMode.label = Suchmodus
+thing-type.config.icalendar.eventfilter.datetimeMode.description = Legt fest, welcher Teil eines Ereignisses in den Suchzeitraum zwischen Beginn und Ende fallen muss
+thing-type.config.icalendar.eventfilter.datetimeMode.option.START = Ereignisse, die im Zeitraum beginnen
+thing-type.config.icalendar.eventfilter.datetimeMode.option.ACTIVE = Ereignisse, die in einer beliebigen Phase des Zeitraums aktiv sind
+thing-type.config.icalendar.eventfilter.datetimeMode.option.END = Ereignisse, die im Zeitraum enden
```

Comment:
Please revert changes to this file. Only the default language is provided by Github, other languages are provided by crowdin.